app/buyers/views/buyers.py

```diff
@@ -0,0 +1,279 @@
+        {'lot': lot['slug']}
+    )
+
+    section = content.get_section(content.get_next_editable_section_id())
+
+    return render_template(
+        "buyers/edit_brief_section.html",
+        framework=framework,
+        data={},
+        section=section,
+        **dict(buyers.config['BASE_TEMPLATE_DATA'])
+    ), 200
+
+
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/create', methods=['POST'])
+def create_new_brief(framework_slug, lot_slug):
+    framework = data_api_client.get_framework(framework_slug)["frameworks"]
+    if framework["status"] != "live":
+        abort(404)
+
+    try:
+        lot = next(lot for lot in framework['lots'] if lot['slug'] == lot_slug)
+    except StopIteration:
+        abort(404)
+    if not lot['allowsBrief']:
+        abort(404)
+
+    content = content_loader.get_manifest(framework_slug, 'edit_brief').filter(
+        {'lot': lot['slug']}
+    )
+
+    section = content.get_section(content.get_next_editable_section_id())
+
+    update_data = section.get_data(request.form)
+
+    try:
+        brief = data_api_client.create_brief(
+            framework_slug,
+            lot_slug,
+            current_user.id,
+            update_data,
+            updated_by=current_user.email_address,
+            page_questions=section.get_field_names()
+        )["briefs"]
+    except HTTPError as e:
+        update_data = section.unformat_data(update_data)
+        errors = section.get_error_messages(e.message, lot_slug)
+
+        return render_template(
+            "buyers/edit_brief_section.html",
+            framework=framework,
+            data=update_data,
+            section=section,
+            errors=errors,
+            **dict(buyers.config['BASE_TEMPLATE_DATA'])
+        ), 400
+
+    return redirect(
+        url_for(".edit_brief_submission",
+                framework_slug=framework_slug,
+                lot_slug=lot_slug,
+                brief_id=brief['id'],
+                section_id=content.get_next_editable_section_id(section.slug)))
+
+
+@buyers.route(
+    '/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/edit/<section_id>',
+    methods=['GET'])
+def edit_brief_submission(framework_slug, lot_slug, brief_id, section_id):
+    framework = data_api_client.get_framework(framework_slug)["frameworks"]
+    if framework["status"] != "live":
+        abort(404)
+
+    try:
+        lot = next(lot for lot in framework['lots'] if lot['slug'] == lot_slug)
+    except StopIteration:
+        abort(404)
+    if not lot['allowsBrief']:
+        abort(404)
+
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+    if not is_brief_associated_with_user(brief, current_user.id) or not brief_can_be_edited(brief):
+        abort(404)
+
+    content = content_loader.get_manifest(framework_slug, 'edit_brief').filter(
+        {'lot': lot['slug']}
+    )
+    section = content.get_section(section_id)
+    if not section:
```

Comment:
On pages dealing with section_ids (especially in POST) we always need to check `section.editable`. In the supplier frontend it's usually done with:

```python
if section is None or not section.editable:
    abort(404
```